Tests/ScipioKitTests/ProjectGeneratorTests.swift

```diff
@@ -8,26 +8,24 @@ private let fixturePath = URL(fileURLWithPath: #file)
     .appendingPathComponent("Resources")
     .appendingPathComponent("Fixtures")
 private let testPackagePath = fixturePath.appendingPathComponent("E2ETestPackage")
+private let clangPackagePath = fixturePath.appendingPathComponent("ClangPackage")
 
 final class ProjectGeneratorTests: XCTestCase {
-    private var projectGenerator: ProjectGenerator!
-    private var package: Package!
     private let fileSystem: some FileSystem = localFileSystem
 
-    override func setUpWithError() throws {
-        package = try Package(packageDirectory: testPackagePath)
-        projectGenerator = ProjectGenerator(package: package,
-                                            buildOptions: .init(buildConfiguration: .debug,
-                                                                isSimulatorSupported: true,
-                                                                isDebugSymbolsEmbedded: false,
-                                                                frameworkType: .static,
-                                                                sdks: [.iOS]),
-                                            fileSystem: localFileSystem)
-
-        try super.setUpWithError()
```

Comment:
Is it deliberate to delete this line?

Reply:
Yes. `setUpWithError` is no longer overridden.